pages.it/windows/whoami.md

```diff
@@ -0,0 +1,28 @@
+# whoami
+
+> Mostra informazioni riguardo all'utente utilizzato.
+> Maggiori informazioni: <https://learn.microsoft.com/it-it/windows-server/administration/windows-commands/whoami>.
```

Comment:
We use the same link otherwise we don't have the resources too keep in sync
```suggestion
> Maggiori informazioni: <https://learn.microsoft.com/windows-server/administration/windows-commands/whoami>.
```

Reply:
Is this something we usually do? It seems reasonable to me to have a link to the *Italian* version of the page.

Reply:
We don't use i18n specifically in MSFT URL because it determinates the user language based on the request headers from the browser. So, in this case the best it's do nothing.

Reply:
> Is this something we usually do? It seems reasonable to me to have a link to the _Italian_ version of the page.

The reason we don't is that the site redirects to the translated version of the page automatically based on your system language or locale setting. It's advantageous because it is controlled by the reader's preference. 

For example let's say somebody's native language is Italian but they prefer a different language to read online (let's say English, etc) in that case it is useful.

Reply:
@kbdharun oh that's neat. Makes sense! Better keep it without `/it-it/` then.

Reply:
Understood, didn't know about the automatic redirect.
Thanks for the suggestion